### ISSUE TITLE
Parsing only root node, but not child nodes (in case of parsing from JSON notation).

### ISSUE THREAD

**Issue by techkrtk** [joaonuno/tree-model-js#63]
Able to store the tree in local storage for persistence using JSON.stringify.
But while retrieving/parsing the JSON data into a variable only the root node is being parsed with the data type of TreeModel, the child nodes are just being parsed as key-value pairs.

Is there a way to store tree persistently if there are more than 100 children?

**Comment by joaonuno** [comment]
Try saving the tree model only `s = JSON.stringify(root.model)` and then parse it back `tree.parse(s)`.

**Comment by techkrtk** [comment]
Yeah, that's what I tried, but no luck, it's just parsing only the root node.

**Comment by techkrtk** [comment]
> Try saving the tree model only `s = JSON.stringify(root.model)` and then parse it back `tree.parse(s)`.

**Comment by techkrtk** [comment]
I tried parsing root. model, even though it's not working.

**Comment by techkrtk** [comment]
I tried parsing root. model, even though it's not working.

**Comment by joaonuno** [comment]
Paste a simple example of what's not working.
